resources/js/src/app/components/itemList/filter/FindologicItemFilter.vue

```diff
@@ -86,7 +90,7 @@
                   :id="'option-' + subCategory.id"
                   class="form-check-input hidden-xs-up"
                   type="checkbox"
-                  :checked="subCategory.selected"
+                  :checked="isCategorySelected(subCategory)"
```

Comment:
Where is `isCategorySelected` defined?

Reply:
Good catch, not necessary anymore to handle cat filter in ItemFilter, we have ItemCategoryDropdown.